### PR TITLE
Adds the Normative Acts ontology (barcelos2013ontology)

### PR DESCRIPTION
**Pull request by pedropaulofb**
All required files for barcelos2013ontology are already available.

### CODE REVIEW

**Review by claudenirmf — CHANGES_REQUESTED** [review]
You must update the names of your files according to the repository's instructions.

We have not required this, but it be nice if the names of the new diagrams matched the names of the original ones.

I like that your screenshots of diagrams kept the figures' labels.

When I tried opening your VP project the diagrams were weird... Can you check whether you committed the most recent version of the project?
image

The package structure of you VP project is inconsistent, please fix that.

**Comment by claudenirmf** [comment]
It would also be nice if your commit titles were clearer about it was done, for example, "adds all required files".

**Comment by claudenirmf** [comment]
Remember to pass the original diagrams through [tinypng](https://tinypng.com/) if you haven't done so.

**Comment by pedropaulofb** [comment]
Thank you for your considerations. I believe that now the data is as
expected.

Kind regards - Cordiali saluti - Atenciosamente,
Pedro Paulo Favato Barcelos


Em sáb., 6 de nov. de 2021 às 17:42, Claudenir Fonseca <
***@***.***> escreveu:

File: barcelos2013ontology/references.bib [hunk 1]
```diff
@@ -0,0 +1,8 @@
+@inproceedings{barcelos2013ontology,
+  title={An Ontology Reference Model for Normative Acts.},
```
Comment (claudenirmf) [inline barcelos2013ontology/references.bib:2]: This `bib` entry is wrong/incomplete. Please double get how to cite CEUR papers (http://ceur-ws.org/Vol-1041/).

**Review by claudenirmf — APPROVED** [review]

**Comment by claudenirmf** [comment]
Thanks for the nice work.

A feedback for next time, though: when finishing your revisions, answer my review confirming your changes for every comment I made. See this answer I got on another PR: https://github.com/unibz-core/ontouml-models/pull/3#issuecomment-967117725

**Pull request merged**